homeassistant/components/sonos/__init__.py

```diff
@@ -325,3 +327,14 @@ async def setup_platforms_and_discovery(self):
                 self.hass, self._async_ssdp_discovered_player, {"st": UPNP_ST}
             )
         )
+
+        self.entry.async_on_unload(
+            self.hass.helpers.event.async_track_time_interval(
```

Comment:
We prefer importing helper names normally instead of accessing `hass.helpers`, to aid typing.